src/diffusers/models/unet_2d_condition.py

```diff
@@ -109,15 +109,15 @@ def __init__(
             "CrossAttnDownBlock2D",
             "DownBlock2D",
         ),
-        mid_block_type: str = "UNetMidBlock2DCrossAttn",
+        mid_block_type: Optional[str] = "UNetMidBlock2DCrossAttn",
```

Comment:
Can we adapt the doc string here correctly?

Reply:
✅